packages/analytics/src/AnalyticsProvider.tsx

```diff
@@ -30,22 +20,32 @@ export interface IAnalyticsProvider {
   ga?: IGAConfig;
   amplitude?: AmplitudeConfig;
   meta?: MetaConfig;
+  disabledFunctions?: Record<string, string[]>;
   children: ReactNode;
 }
 
 const ssr = !(typeof window !== 'undefined' && window.document && window.document.createElement);
 
 const COOKIE_NAME = 'composer_anonymous_id';
 
-const proxyToIntegrations = (integrations: IIntegration[], func: string, args: any[]) => {
-  integrations.forEach((integration) =>
+const proxyToIntegrations = (
+  integrations: Record<string, typeof supportedIntegrations>,
+  func: string,
+  args: any[],
+  disabledFunctions?: Record<string, string[]>,
+) => {
+  Object.keys(integrations).forEach((integrationKey) => {
+    const integration = integrations[integrationKey];
+    if (disabledFunctions && disabledFunctions[integrationKey] && disabledFunctions[integrationKey].includes(func)) {
```

Comment:
is is possible to simplify this?

```suggestion
    if (disabledFunctions?.[integrationKey]?.includes(func)) {
```

Reply:
Yes, it was giving me an error so I used the old school version for now. Fixed.